cypress/cypress/support/e2e.js

```diff
@@ -15,3 +15,19 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands'
+
+var logs = {}
+var num = 0
+
+Cypress.on('log:added', (log) => {
+    if (log.end) {
+        logs[num] = log.message
+        num++
+    }
+})
+
+let name = Cypress.spec.name
+
+after(() => {
+  cy.writeFile('../logs/ui_logs/'+name+'.log.json', logs)
```

Comment:
Thanks for adding it.

We are storing our tests logs in the following path:
logs/submariner/tests_logs/e2e/ - for api tests.
logs/submariner/tests_logs/ui/ - for ui tests.

Here, we are defining the path to the ui logs:
https://github.com/stolostron/acmqe-mcn-test/blob/main/variables#L13

Could you use this variable to provide the path to the logs for ui?

This way we are controlling the path for the logs from a single place.

Reply:
Sure, no problem.
Is the type of the file `.log.json` ok? Or should it be `.yaml`?

Reply:
How is the file looks like?
If it's a simple log file, you could leave it as ...log.

Or does it have json structure?

Reply:
No it does not have json structure.
I will leave it as log.